src/bitwarden_workflow_linter/rules/check_pr_target.py

```diff
@@ -27,16 +27,12 @@ def __init__(self, settings: Optional[Settings] = None, lint_level: Optional[Lin
         self.compatibility = [Workflow]
         self.settings = settings
 
-    def targets_main_branch(self, obj:Workflow) -> bool:
-        if obj.on["pull_request_target"].get("branches"):
-            branches_list = obj.on["pull_request_target"].get("branches")
-            if isinstance(branches_list, str):
-                branches_list = [branches_list]
-            if any(branch != 'main' for branch in branches_list):
-                return False
-        else:
-            return False
-        return True
+    def targets_main_branch(self, obj: Workflow) -> bool:
+        default_branch = self.settings.default_branch if self.settings else "main"
```

Comment:
This should be guaranteed to be set, so in the interest of removing hardcoded values I think it should just be
```suggestion
        default_branch = self.settings.default_branch
```